Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -59,15 +77,25 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
             {
                 searchForFoldersTask.Wait();
 
+                // Identify the unique set of imported folders instead of unique files
+                // in order to avoid duplicating per-folder work.
+                HashSet<string> importedFolders = new HashSet<string>();
                 foreach (string asset in importedAssets.Concat(movedAssets))
                 {
-                    string folder = ResolveFullAssetsPath(asset);
+                    string fullAssetPath = ResolveFullAssetsPath(asset);
```

Comment:
Same question as below, could we pattern match for sentinel files here?

Reply:
Alrighty, posting the resolution in this comment instead of in both. A couple of changes I took here:

1) Make it so that when processing new assets, that we match directly against the regex instead of dropping to folder and then reenumerating. As you mention, this is way better than what I had before.
2) When processing removed assets, fixed that issue where deleting ANY arbitrary file in the root module folder would also cause us to unregister the module folder - now it's only done if the module type matches)